### PR TITLE
feat!: Last Minute Sepolia deployment

### PR DESCRIPTION
**Pull request by utx0**
## Description

* Sepolia deployment 

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

-   [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
-   [X] added `!` to the type prefix if API or client breaking change
-   [ ] ~provided a link to the relevant issue or specification~
-   [X] included the necessary unit and integration tests
-   [X] code is adequately commented
-   [ ] ~updated the relevant documentation or specification~
-   [X] reviewed "Files changed" and left comments if necessary
-   [X] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

-   [x] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
-   [x] confirmed `!` in the type prefix if API or client breaking change
-   [x] confirmed all author checklist items have been addressed
-   [x] reviewed service design and naming
-   [x] reviewed documentation is accurate
-   [x] reviewed tests and test coverage
-   [x] manually tested (if applicable)

### CODE REVIEW

File: script/deploy/DeployProposal.s.sol [hunk 1]
```diff
@@ -2,7 +2,7 @@
 pragma solidity 0.8.18;
 
 import {console} from "@forge-std/console.sol";
-import {zip001 as zip} from "@proposals/zips/zip001.sol"; // zip001 already loaded for next mainNet deployment
+import {zip002 as zip} from "@proposals/zips/zip002.sol"; // zip001 already loaded for next mainNet deployment
```
Comment (utx0) [inline script/deploy/DeployProposal.s.sol]: change this back too `zip001` as well

File: proposals/zips/zip001.sol [hunk 2]
```diff
@@ -62,7 +62,7 @@ contract zip001 is Proposal {
 
     function _afterDeployOnChain(Addresses, address deployer) internal override {
         // Revoke ADMIN role from deployer
-        _core.revokeRole(Roles.ADMIN, deployer);
+        // _core.revokeRole(Roles.ADMIN, deployer); // TODO to be added into next mainNet deployment
```
Comment (utx0) [inline proposals/zips/zip001.sol]: uncomment this

File: proposals/Addresses.sol [hunk 3]
```diff
@@ -59,18 +59,21 @@ contract Addresses is Test {
...
         // 421614: Arbitrum  testNet (sepolia)
-        _addAddress("CORE", 421614, 0x88A6473f76B70472C48cE1C5C6b0ebfcF0FB5A55);
-        _addAddress("GLOBAL_REENTRANCY_LOCK", 421614, 0xB6Bb6Ec96361a17E94991Ba5A28DAa7ca4aC5909);
-        _addAddress("ERC1155_MAX_SUPPLY_MINTABLE_WEARABLES", 421614, 0x0321A813eE91b6a76c3B282a05B35253e4cE75f9);
-        _addAddress("ERC1155_MAX_SUPPLY_MINTABLE_CONSUMABLES", 421614, 0x5F19e60Ba61E230a3e4F8b34444F627F7Dba997C);
-        _addAddress("ERC1155_MAX_SUPPLY_MINTABLE_PLACEABLES", 421614, 0xf9a992a0e7D5C99033c6c256654458E29a296f3B);
-        _addAddress("ERC1155_MAX_SUPPLY_ADMIN_MINTABLE", 421614, 0xE774ccc80519C6F74F7315fCd798E10Bca7Ba53e);
-        _addAddress("ERC1155_AUTO_GRAPH_MINTER", 421614, 0xb2Be249fbf8E3502967Fdff3eB112c744408769A);
-        _addAddress("GAME_CONSUMABLE", 421614, 0x6e6eb0Df27EC2C79488f83aC2D129e9055A9DCa3);
+        // TODO To be updated post deployment
```
Comment (utx0) [inline proposals/Addresses.sol]: remove TODO note

**Review by utx0 — COMMENTED** [review]
Approved

**Review by ghost — APPROVED** [review]

**Review by hosnik — APPROVED** [review]

**Pull request merged**